fig/minted-snippet.tex

```diff
@@ -0,0 +1,24 @@
+\documentclass{article}
+\usepackage{fullpage}
+\usepackage{minted}
+\usepackage{tcolorbox}
+\usepackage{etoolbox}
+\BeforeBeginEnvironment{minted}{\begin{tcolorbox}}%
+\AfterEndEnvironment{minted}{\end{tcolorbox}}%
+
+\begin{document}
+
+\begin{minted}[numbersep=5pt]{python}
+import demes, msprime, moments
+# load the demographic model and specify sample sizes
+model = demes.load("tennessen.yml")
```

Comment:
Oh, I changed the `.yml` file to `.yaml`. I think I read somewhere that the full `.yaml` extension is recommended by the yaml folks.

Reply:
Good point, and I think you're right. Updated.